### ISSUE TITLE
Python scripts out of sync with Workflow tasks, apparently

### ISSUE THREAD

**Issue by lonniev** [smargh/alfred_zotquery#6]
ZotQuery v 10.0 seems to have Alfred Workflow task steps that are out of API sync with the zotquery.py script. For example, all the workflow task steps that call zotquery with the "config" argument fail because the python script expects the argument to be named "configure". (Maybe python allows partial names of arguments as long as they are unambiguous? If so, it doesn't work on this Mac.)

Python on the Mac is a Homebrew 2.7.9 variant.

Furthermore, the export.py script fails to copy found entries into the clipboard. The Notification task is called but the Paste operations are always empty.

**Comment by smargh** [comment]
You are correct. Unfortunately, I got quite busy after I pushed 10.0 and have yet to push a fully working release. There are indeed numerous bugs in the code. I do apologize for that. Do feel free to add a Pull Request tho. But hopefully I can finish up this rewrite over the summer.

**Comment by lonniev** [comment]
If I had a fix for the community and you, yes, I could do a PR for us. Unfortunately, I don't. Meanwhile, is there a last known good release that you recommend I fall back if I do like what ZotQuery promises?

**Comment by smargh** [comment]
The version on [Packal](http://www.packal.org/workflow/zotquery), which is the repository for Alfred Workflows, should always be the most stable. Right now, I think version 8.5 is up there. That is way more stable that 10.0, which is a total rewrite.
